src/main/java/io/github/eocqrs/cmig/meta/Authors.java

```diff
@@ -24,50 +24,63 @@
 
 import com.jcabi.xml.XML;
 import com.jcabi.xml.XMLDocument;
+import org.cactoos.Scalar;
 import org.cactoos.io.ResourceOf;
 
-import java.util.List;
-
 /**
- * File authors.
+ * Authors of State.
  *
  * @author Aliaksei Bialiauski (abialiauski.dev@gmail.com)
  * @since 0.0.0
  */
-public final class Authors implements XpathList {
+public final class Authors implements Scalar<String> {
 
   /**
    * XML.
    */
   private final XML xml;
+  /**
+   * State ID.
+   */
+  private final String id;
 
   /**
    * Ctor.
    *
    * @param doc XML
+   * @param id  State ID
    */
-  public Authors(final XML doc) {
+  public Authors(final XML doc, final String id) {
     this.xml = doc;
+    this.id = id;
   }
 
   /**
    * Ctor.
    *
    * @param name File name
+   * @param id   State ID
    * @throws Exception if something went wrong
    */
-  public Authors(final String name) throws Exception {
+  public Authors(final String name, final String id)
```

Comment:
why is the title in the plural, I don't really understand?

Reply:
@l3r8yJ bug, in prev. implementation was return list of authors